Elysium/src/main/java/hse/elysium/databaseInteractor/TokenService.java

```diff
@@ -3,9 +3,11 @@
 import jakarta.persistence.*;
 
 import hse.elysium.entities.Token;
+import org.springframework.stereotype.Service;
 
 import java.util.ArrayList;
 
+@Service
 public class TokenService {
     private final EntityManagerFactory
         entityManagerFactory = Persistence.createEntityManagerFactory("default");
```

Comment:
Couldn't you achieve it without native SQL queries and with type safety?

Reply:
It's Fedya's class